Automattic/wp-calypso
# Video UI: add CTA according to Figma

This PR adds a CTA to the Video UI according to the [Figma file](EuqfSnWpfYx8fgiBlVmbuA-fi-2266%3A17897).


| Desktop  | Mobile |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/375980/142048767-596099f5-3f68-4ea5-a6c8-d01c90b13f90.png) | ![image](https://user-images.githubusercontent.com/375980/142049033-7795c638-fe29-47b6-8e52-63ab54435076.png)  |

#### Testing instructions

1. Apply this diff to sandbox if not merged: D70171-code
1. Sandbox API
1. Apply this PR
1. In **VideoFooterBar** component in `client/components/videos-ui/index.jsx`:
   * Set `displaySkipLink` to `false`.
   * Set `displayCTA` to `true`.
1. Verify you see the CTA UI display in the bottom of the screen.
2. Verify the track event `calypso_courses_cta_click` is being triggered when clicking the CTA button.


Closes #57506
